# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/ServiceWorkerUtils.h
# Commit: 8fda62a34fb2
# Full Hash: 8fda62a34fb24ee9af0cc61f0d1584f35e64a083
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2025-07-17 09:30:54
# Regressor Bug: 1972117
# File Overlap Count: 1
# Description:
#   Bug 1972117 - Part 3: Move NotificationsCallback to NotificationUtils r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D255860
# ==============================================================================

diff -r f2e7fc3530d1 -r 8fda62a34fb2 dom/serviceworkers/ServiceWorkerUtils.h
--- a/dom/serviceworkers/ServiceWorkerUtils.h	Wed Jul 16 19:52:22 2025 +0000
+++ b/dom/serviceworkers/ServiceWorkerUtils.h	Wed Jul 16 19:52:23 2025 +0000
@@ -39,9 +39,6 @@
 using NavigationPreloadStatePromise =
     MozPromise<IPCNavigationPreloadState, CopyableErrorResult, false>;
 
-using NotificationsPromise =
-    MozPromise<CopyableTArray<IPCNotification>, nsresult, false>;
-
 using ServiceWorkerRegistrationCallback =
     std::function<void(const ServiceWorkerRegistrationDescriptor&)>;
 
